### PR TITLE
Unattended install tests update

### PR DESCRIPTION
**Pull request by teddytpc1**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-jenkins/issues/3138|

## Description
This PR updates the unattended install tests to work with AIO and distributed install.

### CODE REVIEW

File: tests/unattended/install/pytest.ini [hunk 1]
```diff
@@ -6,4 +6,5 @@ markers =
     wazuh_cluster: test for wazuh-clusterd it is meant to be executed on the master node if a wazuh cluster is configured.
     wazuh_worker: test for wazuh cluster worker nodes. It is meant to be executed on the worker nodes.
     elastic: tests to be executed on Elastic hosts.
```
Comment (lastranikos) [inline tests/unattended/install/pytest.ini:8]: This is AIO?

**Review by lastranikos — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]
LGTM

File: tests/unattended/install/pytest.ini (hunk 1 above)
Comment (teddytpc1) [reply tests/unattended/install/pytest.ini:8]: `elastic` is used for every ES host independently of the installation type.

**Pull request merged**